add document for system variable "tidb_enable_fast_table_check" (#13937)
pingcap/docs: system-variables.md

- This variable is used to set whether to enable the statistics `Fast Analyze` feature.
- If the statistics `Fast Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to a non-optimal execution plan, for example, selecting a wrong index. If the execution time of the regular `Analyze` statement is acceptable, it is recommended to disable the `Fast Analyze` feature.

> ### `tidb_enable_fast_table_check` <span class="version-mark">Introduced from v7.2.0 version</span>

Reviewer: ```suggestion
### tidb_enable_fast_table_check <span class="version-mark">New in v7.2.0</span>

```

Reviewer: To be consistent with descriptions of other variables

> **Note:**
>

> > This function does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued indexes) and prefixed indexes.

Reviewer: ```suggestion
> This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued indexes) and prefixed indexes.

```

- Scope: SESSION | GLOBAL

> - Whether to persist to the cluster: Yes

Reviewer: ```suggestion
- Persist to the cluster: Yes
```

Reviewer: To be consistent with descriptions of other variables

- Type: Boolean

> - Default value: `ON

Reviewer: ```suggestion
- Default value: `ON`
```

- This variable is used to control whether to use a checksum-based approach to quickly check the consistency of data and indexes in a table. The default value `ON` means this feature is enabled by default.

> - When turned on, TiDB executes the [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) statement faster.

Reviewer: ```suggestion
- When this variable is enabled, TiDB can execute the [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) statement in a faster way.
```

- This variable is used to set whether to enable the statistics `Fast Analyze` feature.
- If the statistics `Fast Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to a non-optimal execution plan, for example, selecting a wrong index. If the execution time of the regular `Analyze` statement is acceptable, it is recommended to disable the `Fast Analyze` feature.

### tidb_enable_fast_table_check <span class="version-mark">New in v7.2.0</span>

> **Note:**
>

> > This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued-index) and prefixed indexes.

Reviewer 2: ```suggestion
> This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued-index) and prefix indexes.
```

- Scope: SESSION | GLOBAL

> - Persist to the cluster: Yes

Reviewer: ```suggestion
- Persists to the cluster: Yes
```

- Type: Boolean
- Default value: `ON`

> - This variable is used to control whether to use a checksum-based approach to quickly check the consistency of data and indexes in a table. The default value `ON` means this feature is enabled by default.

Reviewer 2: ```suggestion
- This variable is used to control whether to use a checksum-based approach to quickly check the integrity of data and indexes in a table. The default value `ON` means this feature is enabled by default.
```
